kuma/core/models.py

```diff
@@ -34,6 +38,22 @@ def update(self, **kw):
                                           created=False)
 
 
+class IPBan(models.Model):
+    ip = models.GenericIPAddressField()
+    created = models.DateTimeField(default=timezone.now, db_index=True)
+    deleted = models.DateTimeField(null=True, blank=True)
+
+    objects = IPBanManager()
+
+    def delete(self, *args, **kwargs):
+        self.deleted = timezone.now()
+        self.save()
+        IPBanJob.delete(self.ip)
```

Comment:
This needs to be using an instance of a job, so `IPBanJob().delete(self.ip)`. That said, I think we should use the `refresh` method since that not only deletes the cache entry but fills the cache synchronously again right away and doesn't wait till the next request to fill it.